package.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "webgme-executor-worker",
-  "version": "1.0.1",
+  "version": "2.0.0",
```

Comment:
Please don't bump the version in the branch - it'll be done when making a release from master.

Reply:
ok